status_and_problems.tex

```diff
@@ -0,0 +1,134 @@
+
+\begin{itemize}
+\item {\bf Eups}: for the majority of DM packages. This included third party libraries that require customization, or that are not available in the conda channels. The lsst DM binary Eups repository is hosted at \url{https://eups.lsst.codes/}. 
+\item {\bf Sonatype Nexus}: for java based DM software developed packages
+\item {\bf Conda}: for packages public available, that can be used without any customization
+\item {\bf Pip/PiPy}: for python packages public available. In some cases for packages pip/PiPy is used for DM developed python packages, for example \textit{docsteady}.
+\end{itemize}
+
+
+\subsubsection{Distribution System} \label{sec:statusDistrib}
+
+The main tool used for distribution is {\bf Docker}. 
+
+In \url{pipelines.lsst.io} instructions are provided on how to deploy a science pipelines docker image.
+
+A second distribution approach is also provided, using the script \textit{newinstall.sh}. 
+This scripts permits to retrieve a specific version (or reference) of the science pipelines from the Eups repository, and setup the environment for its build.
+
+
+\subsubsection{SW Products Identification} \label{sec:statusIdentification}
+
+The software products are developed using multiple GitHub packages. 
+A GitHub metapackage, for example \textit{lsst\_distrib} for science pipelines, is identifying the software product. 
+All git packages of a software product are dependencies in that metapackage, direct or indirect.
+
+GitHub teams are used to differentiate between two types of git packages:
+
+\begin{itemize}
+\item {\bf Data Management}: it includes all dependencies of \textit{lsst\_distrib}
+\item {\bf DM Externals}: for packages used to build and distribute \textit{lsst\_distrib} but that they are not part of it.
+\end{itemize}
+
+There is a third team, {\bf DM Auxiliary}, that identify auxiliary packages to be tagged, but that they are not part of the software product:
+
+Note that in reality, \textit{lsst\_distrib} do not correspond to a software product, but to a distribution product.
+It includes not only the different pipelines, but also test data and build tools.
+
+
+\subsubsection{Environment} \label{sec:statusEnvs}
+
+The conda environment to be used for building the science pipelines is defined in the Git package \textit{scipipe\_conda\_env}.
+No releases nor tags are made  on the conda environment git package.
+
+\textit{TO BE CLARIFIED: how are java build environments managed?}
+
+
+\subsubsection{Build Tools} \label{sec:statusTools}
+
+The build tools used to build science pipelines software are available in \textit{lsstsw} and \textit{lsst\_build} git packages.
+
+The Jenkins scripts also are an important part of the tooling, facilitating a large quantity of actions, like, test a ticket branch, automate releases, or provide periodic build (weekly/daily).
+
+The tools picture is completed by \textit{codekit}, that permits to interact with multiple GitHub packages at the same time.
```

Comment:
The tools picture is completed by \textit{codekit}, which  permits interaction with multiple GitHub packages.

(But not clear what you meant )

Reply:
I added some text in the example.